packages/lib/src/treeUtils/sandbox.ts

```diff
@@ -222,7 +229,9 @@ export class SandboxManager {
           for (let i = 0; i < len; i++) {
             patches.push(...recorder.events[i].patches)
           }
+          this.isComitting = true
```

Comment:
Save the old value then restore the value in a finally block to ensure a failing apply won't break it

Reply:
The saving is to make sure if for some reason two calls of this get into the stack it will be restored properly

Reply:
Good idea, done in 31869e5.